suite-native/accounts/src/components/StakingItem.tsx

```diff
@@ -0,0 +1,61 @@
+import { TouchableOpacity } from 'react-native';
+
+import { Box, RoundedIcon, Text } from '@suite-native/atoms';
+import { CryptoAmountFormatter, CryptoToFiatAmountFormatter } from '@suite-native/formatters';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Account } from '@suite-common/wallet-types';
+import {
+    getAccountTotalStakingBalance,
+    getAccountTotalStakingBalanceWei,
+} from '@suite-common/wallet-utils';
+import { Translation } from '@suite-native/intl';
+
+const valuesContainerStyle = prepareNativeStyle(utils => ({
+    maxWidth: '40%',
+    flexShrink: 0,
+    alignItems: 'flex-end',
+    paddingLeft: utils.spacings.small,
+}));
+
+const stakingItemStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingVertical: 12,
+    paddingHorizontal: utils.spacings.medium,
+}));
+
+type StakingItemProps = {
+    account: Account;
+};
+
+export const StakingItem = ({ account }: StakingItemProps) => {
```

Comment:
This component should use `AccountListItemBase` or this component shoudn't exist at all and `AccountListItemBase` could be used directly

Reply:
done. This component has been deleted. https://github.com/trezor/trezor-suite/pull/14072/commits/565eadf4d973e9eded8dc5caa0e6da4d7dc37733